terraform/environments/mlra/main.tf

```diff
@@ -49,14 +49,29 @@ resource "aws_lb_listener" "alb_listener" {
   default_action {
     type             = "forward"
     target_group_arn = aws_lb_target_group.alb_target_group.arn
+    #TODO cloudfront rule may still need to be applied. pending cutover strategy decisions
```

Comment:
Know this was an example I sent, but can we place this comment somewhere where it makes more sense, and perhaps elaborate on why the listener rule is, at this point, not been provisioned?

Reply:
I have modified the comment, and put it in the section below the default_action